### PR TITLE
Fix inspection errors

### PR DESCRIPTION
**Pull request by whitewindmills**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/customresourceinterpreter/webhook/app/workloadwebhook.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type workloadInterpreter struct {
 
 // Handle implements interpreter.Handler interface.
 // It yields a response to an ExploreRequest.
-func (e *workloadInterpreter) Handle(ctx context.Context, req interpreter.Request) interpreter.Response {
+func (e *workloadInterpreter) Handle(_ context.Context, req interpreter.Request) interpreter.Response {
```
Comment (chaunceyjiang) [inline examples/customresourceinterpreter/webhook/app/workloadwebhook.go:29]: How did you find these lint errors, what tools did you use?

Comment (whitewindmills) [reply examples/customresourceinterpreter/webhook/app/workloadwebhook.go:29]: It's [GoLand Inspection](https://www.jetbrains.com/help/go/command-line-code-inspector.html#494cfa25). I don't know what tools it integrates, but it's very easy to find these lint errors.

**Review by zishen — COMMENTED** [review]
/lgtm
ok

**Review by RainbowMango — COMMENTED** [review]
/assign

File: examples/customresourceinterpreter/webhook/app/options/options.go [hunk 2]
```diff
@@ -23,8 +23,8 @@ func NewOptions() *Options {
 }
 
 // AddFlags adds flags to the specified FlagSet.
-func (o *Options) AddFlags(flags *pflag.FlagSet) {
-	flags.StringVar(&o.BindAddress, "bind-address", defaultBindAddress, "The IP address on which to listen for the --secure-port port.")
-	flags.IntVar(&o.SecurePort, "secure-port", defaultPort, "The secure port on which to serve HTTPS.")
-	flags.StringVar(&o.CertDir, "cert-dir", defaultCertDir, "The directory that contains the server key(named tls.key) and certificate(named tls.crt).")
+func (opts *Options) AddFlags(flags *pflag.FlagSet) {
```
Comment (RainbowMango) [inline examples/customresourceinterpreter/webhook/app/options/options.go:26]: Why not use `o`?

File: operator/pkg/util/apiclient/idempotency.go [hunk 3]
```diff
@@ -231,7 +231,7 @@ func CreateOrUpdateStatefulSet(client clientset.Interface, statefuleSet *appsv1.
 }
 
 // DeleteDeploymentIfHasLabels delete a Deployment that exists the given labels.
-func DeleteDeploymentIfHasLabels(client clientset.Interface, name, namespace string, ls labels.Labels) error {
+func DeleteDeploymentIfHasLabels(client clientset.Interface, name, namespace string, _ labels.Labels) error {
```
Comment (RainbowMango) [inline operator/pkg/util/apiclient/idempotency.go:234]: Please operator owners consider if we need to remove the `ls labels.Labels` and update the function name.
@Poor12 @calvin0327 @lonelyCZ

File: pkg/karmadactl/cmdinit/karmada/rbac_test.go [hunk 4]
```diff
@@ -6,18 +6,16 @@ import (
...
 )
 
-var noError = false
-
 func Test_grantProxyPermissionToAdmin(t *testing.T) {
 	client := fake.NewSimpleClientset()
-	if err := grantProxyPermissionToAdmin(client); (err != nil) != noError {
-		t.Errorf("grantProxyPermissionToAdmin() error = %v, wantErr %v", err, noError)
+	if err := grantProxyPermissionToAdmin(client); err != nil {
+		t.Errorf("grantProxyPermissionToAdmin() expected no error, but got err: %v", err)
```
Comment (RainbowMango) [inline pkg/karmadactl/cmdinit/karmada/rbac_test.go:12]: nice finding! 
cc @jwcesign

File: pkg/karmadactl/get/get.go [hunk 5]
```diff
@@ -358,7 +358,7 @@ func (g *CommandGetOptions) Run(f util.Factory, cmd *cobra.Command, args []strin
 }
 
 // printObjs print objects in multi clusters
-func (g *CommandGetOptions) printObjs(objs []Obj, allErrs *[]error, args []string) {
+func (g *CommandGetOptions) printObjs(objs []Obj, allErrs *[]error, _ []string) {
```
Comment (RainbowMango) [inline pkg/karmadactl/get/get.go:361]: Can we remove the `args []string`? @lonelyCZ

File: pkg/modeling/modeling.go [hunk 6]
```diff
@@ -63,7 +63,10 @@ type ResourceList map[clusterapis.ResourceName]resource.Quantity
 
 // InitSummary is the init function of modeling data structure
 func InitSummary(resourceModels []clusterapis.ResourceModel) (ResourceSummary, error) {
-	rsName, rsList := []clusterapis.ResourceName{}, []ResourceList{}
+	var (
+		rsName []clusterapis.ResourceName
+		rsList []ResourceList
+	)
```
Comment (RainbowMango) [inline pkg/modeling/modeling.go:69]:
```suggestion
	var rsName []clusterapis.ResourceName
	var rsList []ResourceList
```
We don't usually use variable blocks.

File: pkg/modeling/modeling_test.go [hunk 7]
```diff
@@ -6,7 +6,7 @@ import (
 	"reflect"
 	"testing"
 
-	rbt "github.com/emirpasic/gods/trees/redblacktree"
+	rbtree "github.com/emirpasic/gods/trees/redblacktree"
```
Comment (RainbowMango) [inline pkg/modeling/modeling_test.go:9]:
```suggestion
	"github.com/emirpasic/gods/trees/redblacktree"
```
It's ok to omit the alias.

File: pkg/util/lifted/corevalidation.go [hunk 8]
```diff
@@ -36,7 +36,7 @@ import (
 )
 
 // +lifted:source=https://github.com/kubernetes/kubernetes/blob/release-1.22/pkg/apis/core/validation/validation.go#L59
-const isNegativeErrorMsg string = apimachineryvalidation.IsNegativeErrorMsg
+const isNegativeErrorMsg = apimachineryvalidation.IsNegativeErrorMsg
```
Comment (RainbowMango) [inline pkg/util/lifted/corevalidation.go:39]: Please don't touch the `lifted` code if it is not a bug.

Comment (RainbowMango) [reply pkg/util/lifted/corevalidation.go:39]: Same as below.

File: examples/customresourceinterpreter/webhook/app/options/options.go (hunk 2 above)
Comment (whitewindmills) [reply examples/customresourceinterpreter/webhook/app/options/options.go:26]: > Why not use `o`?

Receiver names are different.
https://github.com/karmada-io/karmada/blob/master/examples/customresourceinterpreter/webhook/app/options/validation.go#L10

File: pkg/modeling/modeling.go (hunk 6 above)
Comment (whitewindmills) [reply pkg/modeling/modeling.go:69]: Got it.

File: pkg/modeling/modeling_test.go (hunk 7 above)
Comment (whitewindmills) [reply pkg/modeling/modeling_test.go:9]: Got it.

File: pkg/util/lifted/corevalidation.go (hunk 8 above)
Comment (whitewindmills) [reply pkg/util/lifted/corevalidation.go:39]: Got it.

File: examples/customresourceinterpreter/webhook/app/options/options.go (hunk 2 above)
Comment (RainbowMango) [reply examples/customresourceinterpreter/webhook/app/options/options.go:26]: I know, I tend to `o` compared to `opts`, more shorter :)

Comment (whitewindmills) [reply examples/customresourceinterpreter/webhook/app/options/options.go:26]: > I know, I tend to `o` compared to `opts`, more shorter :)

Okay, it does make sense to me.

File: operator/pkg/util/apiclient/idempotency.go (hunk 3 above)
Comment (Poor12) [reply operator/pkg/util/apiclient/idempotency.go:234]: I'd like to delete this parameter. It seems that this parameter has no room for expansion. /cc @calvin0327

**Review by whitewindmills — COMMENTED** [review]
/cc @RainbowMango @Poor12 @lonelyCZ @jwcesign for checking

**Comment by RainbowMango** [comment]
I remember we are blocked by the operator code, right? Please @Poor12 take a look.

**Comment by Poor12** [comment]
> I remember we are blocked by the operator code, right? Please @Poor12 take a look.

I'm ok with changes in karmada-operator.
/lgtm

**Pull request merged**